schemas/date.schema.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+
+  "type":"string",
+  "pattern":"^\d{4}-((0[1-9])|(1[012]))-((0[1-9]|[12]\d)|3[01])$"
```

Comment:
Do we need all those brackets here?
what about pretty simple: `^\d{4}-[0-1]\d-[0-3]\d$`